README.md

```diff
@@ -2,11 +2,11 @@
 
 The Lightstep Metrics Plugin allows you to view metrics from Lightstep directly in Grafana. With minimal effort, you can use the plugin to continue viewing your existing visualizations in Grafana, using Lightstep as the data source. Then when you notice a deviation, you can click in a chart in Grafana and navigate into Lightstep to use [Change Intelligence](https://docs.lightstep.com/docs/investigate-metric-deviation) and uncover the root cause.
 
-![Start Change Intelligence from Grafana](https://github.com/lightstep/lightstep-metrics-datasource/raw/main/images/docs/graf_metciStart.png)
+![Start Change Intelligence from Grafana](/images/docs/graf_metciStart.png)
```

Comment:
🤔 I think if we use relative paths here the images won't always load (e.g. if I clone the repo and open this document).

Is there another reason to use relative paths that I'm missing?

Reply:
I have no idea why this keeps happening. I used relative links (I SWEAR I did!), but this is the second time git changed them to absolute. Any ideas how/why?

Reply:
Here's a screenshot of what I have locally:
![image](https://user-images.githubusercontent.com/50384513/118730760-f6759d00-b7ec-11eb-97ea-aa5f96b7de3b.png)

Reply:
Ah maybe I misunderstood, and I see now that Github [recommends relative image paths](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-readmes#relative-links-and-image-paths-in-readme-files) 👍 .

I'm not sure we want the leading forward slash, though. If we removed it or changed it to `./images...`, I think the images would load more reliably.

As for git changing things, well, that's odd 🤔 . I'd recommend running `yarn build` before committing so the files in `dist` include all your changes. Maybe that will help! I'm happy to take this offline too 😄

Reply:
oh i did not know this!

> GitHub will automatically transform your relative link or image path based on whatever branch you're currently on, so that the link or path always works. You can use all relative link operands, such as ./ and ../.